src/router/legacyPermalinkManagement.routerPlugin.js

```diff
@@ -58,9 +57,7 @@ const handleLegacyParams = (legacyParams, store, to, next) => {
             case 'zoom':
                 value = translateSwisstopoPyramidZoomToMercatorZoom(legacyParams[param])
                 if (!value) {
-                    // if the value is not defined in the old zoom system, we use the 'default' zoom level
-                    // of 8 (which will roughly show the whole territory of Switzerland)
-                    value = 8
+                    value = DEFAULT_PROJECTION.defaultZoom
```

Comment:
Should we not here translate the zoom to the correct projection based on the store `projection` value ? Also at line 58 we use mercator but line 60 might be lv95, depending on the default projection.

Reply:
you are totally right, and I'm working on that for a different PR right now as it's a bit extensive how much code must change to accommodate that...